src/main/java/net/minecraftforge/common/data/ExistingFileHelper.java

```diff
@@ -48,17 +52,24 @@
     private final SimpleReloadableResourceManager clientResources, serverData;
     private final boolean enable;
 
-    public ExistingFileHelper(Collection<Path> existingPacks, boolean enable) {
+    public ExistingFileHelper(Collection<Path> existingPacks, @Nullable final String assetIndex, @Nullable final File assetsDir, boolean enable) {
```

Comment:
What is your facination with adding arguments to the middle of the argument list?
New arguments go on the end, and ideally keep the old method bouncing to the new with default values.